website/docs/en/blog/announcing-0.6.mdx

```diff
@@ -0,0 +1,40 @@
+---
+date: 2024-04-10 16:00:00
+---
+
+import { PackageManagerTabs } from '@theme';
+
+# Rspack 0.6 Release Announcement
+
+> April 10, 2024
+
+## Major Feature Updates
+
+### Built-in support for mini-css-extract-plugin
+
+## Breaking Changes
+
+### Enable new tree shaking by default
```

Comment:
The new tree shaking can be a `Major Feature Updates` rather than `Breaking Changes`